Makefile

```diff
@@ -26,8 +26,8 @@ envinit:
 	GO111MODULE=on go get -u github.com/vektra/mockery/...
 	GO111MODULE=off go get -u github.com/matryer/moq
 	GO111MODULE=off go get golang.org/x/tools/cmd/goimports
```

Comment:
maybe the ones above should also use `go install` instead of `go get` :thinking: